docs/portal-contribute.md

```diff
@@ -46,12 +46,34 @@ If you think something is wrong or something fundamental should change, discussi
 * [Issues in the GitHub Repo](https://github.com/cardano-foundation/developer-portal/issues) 
 * [Discussions in the GitHub Repo](https://github.com/cardano-foundation/developer-portal/discussions)
 
-### Add projects to showcase or builder tools
-Help to add good projects or builder tools. Please note the different guidelines for [adding showcases](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/showcases.js) or [builder tools](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/builder-tools.js).
+### Add a project to showcase 
+Help to add good projects to the [showcase section](http://developers.cardano.org/showcase/). Please note the guidelines for adding new projects:
+* It must be built on Cardano and have a real use case. For example, a forum where people can talk about Cardano is great, but nothing for this showcase section.
```

Comment:
```suggestion
* It must be built on Cardano and have a real use case. For example, a forum where people can talk about Cardano is great, but not for this showcase section.
```
Have I understood this correctly? (i.e. a Forum not implemented in Cardano will _not_ be great for the Showcase section and therefore we will have _nothing_ of this kind here)